src/data/analysis/IOTM Reviews/Chest Mimic.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: "Chest Mimic: Overview"
+date: "2024-01-03"
+slug: "2024-01-chest-mimic"
+author: "Captain Scotch"
+section: "analysis"
+tag: "IOTM Overview"
+---
+With the backup camera leaving standard, many speedsters were wondering if we'd get a new copy source at some point in the coming year; we have one source (sort of) through the Recall Habitats skill from the Book of Facts, but that often left you with extra copies (i.e., copy one NSA, get the two more you need, you have 3 more stupid ones floating around you didn't need). Well! We can safely say that we have *ample* access to copies now. Let's talk about the reason for our change in fortune -- January's **Chest Mimic**.
+
+## General Summary
+
+In a nice newbie-friendly boost, the Chest Mimic is a fairychaun -- that is, a combination of a Baby Gravy Fairy (which helps item drop) and a Leprechaun (which helps meat drop). These are, broadly speaking, the two most desirable familiar enchantments; funnily enough, there's already a slightly better base-fam-value fairychaun available in standard with the Jill-of-all-Trades, but the Chest Mimic being a second one should help folks who missed out on the Jill save their meat for more important IOTMs. 
+
+This is a little bit of a tangent, but as I discussed in our <Link to="/analysis/2023-10-jill/">Jill-of-all-Trades article</Link>, the availability of base fairychauns in the IOTM familiar suite is both a neat boost to newbies and a funny example of how powercreep has gradually impacted the game. Up until 2015, the only "true" fairychauns available were through a yearly Familiar of the Year that cost two Mr. Accessories; since then, we've gotten a handful of fairychauns that cost just one, starting with 2018's Cat Burglar (a familiar that was markedly better than any of the fairychauns that came before it), continuing with 2023's Jill-of-All-Trades (discussed previously), and continuing less than 4 months later with the Chest Mimic. As noted in the Jill article, the funniest part about this is that this really does almost solely help newbies -- in high-end speedplay, familiars are used much more for their core turnsaving effects than any focus whatsoever on their core familiar function. Just kind of funny. Being a fairychaun is barely a footnote anymore!
```

Comment:
Do we want this at the top of the post in the summary? Might be worth having a Tangent section.